> So integration tests, i'm getting these two errors, are they what you are getting.
> 
> 1. Unexpected error creating GOGS repo: Repository creation call did not return the expected value (returned 401)
> 2. Exception caught during execution of push command
>    -> Source ref refs/heads/master doesn't resolve to any object.

I get the following output (running Java 8u322 on Debian testing with Docker ):

```
Running org.jenkinsci.plugins.gogs.GogsWebHook_IT
Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 12.463 sec <<< FAILURE!
smokeTest_build_testBranch(org.jenkinsci.plugins.gogs.GogsWebHook_IT)  Time elapsed: 5.351 sec  <<< FAILURE!
java.lang.AssertionError: Unexpected error creating GOGS repo: Repository creation call did not return the expected value (returned 401)
        at org.junit.Assert.fail(Assert.java:89)
        at org.jenkinsci.plugins.gogs.GogsWebHook_IT.doItTest(GogsWebHook_IT.java:121)
        at org.jenkinsci.plugins.gogs.GogsWebHook_IT.smokeTest_build_testBranch(GogsWebHook_IT.java:84)
...
smokeTest_build_masterBranch(org.jenkinsci.plugins.gogs.GogsWebHook_IT)  Time elapsed: 7.077 sec  <<< ERROR!
org.apache.http.client.HttpResponseException: status code: 403, reason phrase: Forbidden
        at com.offbytwo.jenkins.client.validator.HttpResponseValidator.validateResponse(HttpResponseValidator.java:11)
        at com.offbytwo.jenkins.client.JenkinsHttpClient.post_xml(JenkinsHttpClient.java:342)
        at com.offbytwo.jenkins.JenkinsServer.createJob(JenkinsServer.java:382)
        at com.offbytwo.jenkins.JenkinsServer.createJob(JenkinsServer.java:338)
        at org.jenkinsci.plugins.gogs.GogsWebHook_IT.doItTest(GogsWebHook_IT.java:174)
        at org.jenkinsci.plugins.gogs.GogsWebHook_IT.smokeTest_build_masterBranch(GogsWebHook_IT.java:74)

Results :

Failed tests:
  GogsWebHook_IT.smokeTest_build_testBranch:84->doItTest:121 Unexpected error creating GOGS repo: Repository creation call did not return the expected value (returned 401)

Tests in error:
  GogsWebHook_IT.smokeTest_build_masterBranch:74->doItTest:174 » HttpResponse st...

Tests run: 2, Failures: 1, Errors: 1, Skipped: 0
```

--- a/pom.xml
+++ b/pom.xml
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.11</version>
+    <version>4.34</version>
   </parent>
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>gogs-webhook</artifactId>

--- a/pom.xml
+++ b/pom.xml
@@ -37,6 +37,18 @@
     </developer>
   </developers>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.289.x</artifactId>
+        <version>961.vf0c9f6f59827</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>

--- a/pom.xml
+++ b/pom.xml
@@ -14,7 +14,6 @@
   <properties>
     <jenkins.version>1.625.3</jenkins.version>
     <java.level>8</java.level>
-    <jenkins-test-harness.version>2.13</jenkins-test-harness.version>
     <java.level.test>8</java.level.test>
     <findbugs.failOnError>false</findbugs.failOnError>
   </properties>

